resources/views/site/visit.blade.php

```diff
@@ -8,73 +8,75 @@
-                </div>
-                <div class="col-sm-3">
-                    {!! Form::label('rating', 'Rating') !!}
-                    {!! Form::select('rating', [
-                        1 => 'Observer (OBS)', 2 => 'Student 1 (S1)',
-                        3 => 'Student 2 (S2)', 4 => 'Senior Student (S3)',
-                        5 => 'Controller (C1)', 7 => 'Senior Controller (C3)',
-                        8 => 'Instructor (I1)', 10 => 'Senior Instructor (I3)'
-                    ], null, ['placeholder' => 'Select Rating', 'class' => 'form-control']) !!}
-                </div>
-                <div class="col-sm-3">
-                    {!! Form::label('home', 'Home ARTCC') !!}
-                    {!! Form::text('home', null, ['placeholder' => 'Required', 'class' => 'form-control']) !!}
-                </div>
+            <div class="col-sm-6">
+                {!! Form::label('name', 'Full Name') !!}
+                {!! Form::text('name', null, ['placeholder' => 'Required', 'class' => 'form-control']) !!}
             </div>
         </div>
-        <div class="form-group">
-            {!! Form::label('reason', 'Explanation of Why You Want to Visit the ZTL ARTCC') !!}
-            {!! Form::textArea('reason', null, ['placeholder' => 'Required', 'class' => 'form-control']) !!}
+    </div>
+    <div class="form-group">
+        <div class="row">
+            <div class="col-sm-6">
+                {!! Form::label('email', 'Email') !!}
+                {!! Form::email('email', null, ['placeholder' => 'Required', 'class' => 'form-control']) !!}
+            </div>
+            <div class="col-sm-3">
+                {!! Form::label('rating', 'Rating') !!}
+                {!! Form::select('rating', [
+                1 => 'Observer (OBS)', 2 => 'Student 1 (S1)',
+                3 => 'Student 2 (S2)', 4 => 'Senior Student (S3)',
+                5 => 'Controller (C1)', 7 => 'Senior Controller (C3)',
+                8 => 'Instructor (I1)', 10 => 'Senior Instructor (I3)'
+                ], null, ['placeholder' => 'Select Rating', 'class' => 'form-control']) !!}
+            </div>
+            <div class="col-sm-3">
+                {!! Form::label('home', 'Home ARTCC') !!}
+                {!! Form::text('home', null, ['placeholder' => 'Required', 'class' => 'form-control']) !!}
+            </div>
         </div>
-        <div class="g-recaptcha" data-sitekey="6LcC3XoUAAAAAG8ST6HXqS3_reIZRLcA09sDdodw"></div>
-        <br>
-        <button class="btn btn-success" type="submit">Submit</button>
+    </div>
+    <div class="form-group">
+        {!! Form::label('reason', 'Explanation of Why You Want to Visit the ZTL ARTCC') !!}
+        {!! Form::textArea('reason', null, ['placeholder' => 'Required', 'class' => 'form-control']) !!}
+    </div>
+    <div class="g-recaptcha" data-sitekey="6LcC3XoUAAAAAG8ST6HXqS3_reIZRLcA09sDdodw"></div>
```

Comment:
Is it possible to move this key into an environment variable? Even though it might not be private, I bet it's used in other places so it is a good idea to have one place for it.

Reply:
Valid, but out-of-scope for this PR. Implemented in #568